python/ray/train/_internal/data_config.py

```diff
@@ -46,6 +47,18 @@ def __init__(
             execution_options or DataConfig.default_ingest_options()
         )
 
+        self._num_train_cpus = 0.0
+        self._num_train_gpus = 0.0
+
+    def set_train_total_resources(self, num_train_cpus: float, num_train_gpus: float):
```

Comment:
Shall we leave a TODO here, since we only considered CPU and GPU here. Maybe in the future Ray Train & Data will use other resource types, then we may also exclude it here.